lib/scope.js

```diff
@@ -110,15 +114,11 @@ AST_Toplevel.DEFMETHOD("figure_out_scope", function(options){
     var in_destructuring = null;
     var in_export;
     var tw = new TreeWalker(function(node, descend){
-        var create_a_block_scope =
-            (options.screw_ie8 && node instanceof AST_Catch) ||
```

Comment:
Fix in #1344 as compression testing setup was to blame for different output results. screw_ie8 dependent behaviour is done on AST_SymbolCatch.